tsdb/config.go

```diff
@@ -57,6 +60,9 @@ type Config struct {
 	CompactFullWriteColdDuration   toml.Duration `toml:"compact-full-write-cold-duration"`
 	MaxPointsPerBlock              int           `toml:"max-points-per-block"`
 
+	// Limits
+	MaxSeriesPerDatabase int `toml:"max-series-per-database"`
```

Comment:
Can you add docs that describe the behavior of this option and and that `0` disables it.  Something like:

 MaxSeriesPerDatabase is the maximum number of series a node hold per database. When this limit is exceeded, writes return a `max series per database exceeded` error.  A value of `0` disables the limit.